<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WVipDlg</class>
 <widget class="QWidget" name="WVipDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>500</width>
     <height>611</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="frameStage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>605</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>605</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>172</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>165</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Vip属性</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Vip   名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbVipId">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>有 效 性：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbVipEnable">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>有效</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>无效</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>控制方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbVipCtrl">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>自动</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>手动</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>方    向：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbVipDir">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>上行</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>下行</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>开始时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="edVipStart">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="edVipEnd">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDel">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnApp">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>应用</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="readBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="appBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>168</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>165</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>节点参数</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>节 点 名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbNodeName">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>节点阶段：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbStage">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>时 间 差：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbTimeOffset">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="rbDir2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-&lt;--&lt;---下行</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbDir1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上行---&gt;--&gt;-&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QCheckBox" name="cbVipEnable_2">
      <property name="text">
       <string>Vip路线有效</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbManual">
      <property name="text">
       <string>手动</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbDir">
      <property name="text">
       <string>上行</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>13</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>启动Vip路线</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbVipStateCheck">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>状态检测</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layoutContent">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameContent">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
